### DISCUSSION TITLE
[Explore] Save and Save As experience on chart explore

### DISCUSSION THREAD

**Discussion by zuzana-vej** [apache/superset#18535] [Ideas]
**Is your feature request related to a problem? Please describe.**
Some users are frustrated about the save and save as buttons on charts, and feel like the current experience is not very straightforward - when chart is overriden vs. saved as a new chart. 
Specifically what is not clear to some users is:

- If they use "save" but change the name, it's not clear that they are overwriting the chart but with a new name

**Describe the solution you'd like**
How about using "`Save as ... (new chart)`" & make it clear whether same name is allowed or not

If they chose "save" but change name, maybe letting them know that chart will be overwritten but with a new name or adding a note about this (can be a popover next to the red asterix)

<img width="479" alt="Screen Shot 2021-12-08 at 5 27 13 PM" src="https://user-images.githubusercontent.com/61221714/145317765-a58221b0-3561-4d02-bda9-5b4a5b0f1aea.png">

**Comment by geido** [comment]
Thanks for the suggestion @zuzana-vej!
